docs/blog/vitest-3.2.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Vitest 3.2 is out!
+author:
+  name: The Vitest Team
+date: 2025-01-17
+sidebar: false
+head:
+  - - meta
+    - property: og:type
+      content: website
+  - - meta
+    - property: og:title
+      content: Announcing Vitest 3.2
+  - - meta
+    - property: og:image
+      content: https://vitest.dev/og-vitest-3.jpg
+  - - meta
+    - property: og:url
+      content: https://vitest.dev/blog/vitest-3
+  - - meta
+    - property: og:description
+      content: Vitest 3.2 Release Announcement
+  - - meta
+    - name: twitter:card
+      content: summary_large_image
+---
+
+Vitest 3.2 is out!
+
+_Date Day, 2025_
+
+Vitest 3.2 focuses on improvements to Browser Mode and TypeScript support. This release also includes some new useful methods, config options and deprecates the `workspace` config in favour of `projects`.
+
+## `workspace` is Deprecated
+
+In an effort to simplify the configuration, the team decided to deprecate the separate `vitest.workspace` file and recommend using only the `projects` option in the root config. This also simplifies how the global options are configured (because you don't need to guess how to add reporters when you have no root config).
+
+We also decided to deprecate the `workspace` name because it clashes with other tools like PNPM that provide monorepo support via this option. Vitest doesn't run these projects with separate `CWD` and treats them more like sub-Vitests. It also gives us more space to come up with a better solution for monorepos without breaking others.
+
```

Comment:
I think we should add small example here:

````suggestion

```diff
import { defineConfig } from "vitest/config";

export default defineConfig({
  test: {
    // "test.workspace" is now "test.projects"
-    workspace: [
+    projects: [
      { test: { name: "Unit" } },
      { test: { name: "Integration" } },
    ],
  },
});
```

````